lib/spices/helpers/wait.rb

```
@@ -7,6 +7,7 @@ def self.for(timeout=15, &_block)
 
   def self.for_ui_animation_to_end
     Spices.world.wait_for_none_animating(timeout: 10, screenshot_on_error: false)
-  rescue
+  rescue => error
+    raise unless error.message.include? "undefined method `wait_for_none_animating'"
```

I didn't wanted the log there, but if I just deleted the log rubo would complain about suppressing teh exception. I figured that since we're waiting for a specific a exception to take place, we should raise an error if some other exception is actually thrown.